.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -10,13 +10,12 @@ Closes #
 -
 
 ### Checklist
-- [ ] My code follows the [code-style](https://github.com/GIScience/ohsome-api/blob/master/CONTRIBUTING.md) rules, and I have checked on the [static analyses](https://jenkins.ohsome.org/job/oshdb/view/change-requests/) and [benchmark](https://reports.ohsome.org/oshdb-benchmarks/) (if applicable) results
+- [ ] My code follows the [code-style](https://github.com/GIScience/ohsome-api/blob/master/CONTRIBUTING.md) rules, and I have checked on the [static analyses](https://jenkins.ohsome.org/job/ohsome-api/view/change-requests/)
 - [ ] I have commented my code
 - [ ] I have written javadoc (required for public methods)
 - [ ] I have added sufficient unit tests
-- [ ] I have made corresponding changes to the [documentation](https://github.com/GIScience/oshdb/tree/master/documentation)
+- [ ] I have made corresponding changes to the [documentation](https://github.com/GIScience/ohsome-api/tree/master/docs)
 - [ ] I have updated the [CHANGELOG.md](https://github.com/GIScience/oshdb/blob/master/CHANGELOG.md)
-- [ ] I have adjusted the [examples](https://gitlab.gistools.geog.uni-heidelberg.de/giscience/big-data/ohsome/oshdb-examples) or [created an issue](https://gitlab.gistools.geog.uni-heidelberg.de/giscience/big-data/ohsome/oshdb-examples/-/issues/new) in the corresponding repository
-- [ ] I have adjusted the [benchmark](https://reports.ohsome.org/oshdb-benchmarks/) or [created an issue](https://gitlab.gistools.geog.uni-heidelberg.de/giscience/big-data/ohsome/oshdb-benchmarks/-/issues/new) in the corresponding repository
+- [ ] I have adjusted the examples in the [check-ohsome-api](https://gitlab.gistools.geog.uni-heidelberg.de/giscience/big-data/ohsome/helpers/check-ohsome-api) script or [created an issue](https://gitlab.gistools.geog.uni-heidelberg.de/giscience/big-data/ohsome/helpers/check-ohsome-api/-/issues/new) in the corresponding repository
```

Comment:
Instead of examples I used the `check-ohsome-api` repo here.

Reply:
With this, you require some python skills then from the contributor as well, right?

Reply:
No, the Idea is, that people dump their requests (in whatever format, e.g. cURL, python-requests, …) as issue in this repo and we decide if we need to integrate them.

Reply:
Preferable directly write the request as Merge Request in JSON format, but that's something the person doesn't necessarily need to do. E.g.:
```json
{
    "requests": [
        {
            "method": "POST",
            "path": "/elements/count",
            "parameters": {
                "time": "2010-01-01/2016-08-01/P2Y2M2D",
                "filter": "highway=residential and type:way",
                "note": "bboxes missing on purpose"
            }
        },
        {
            "method": "POST",
            "path": "/elements/count",
            "parameters": {
                "bboxes": "8.6561,49.3634,8.7145,49.4344",
                "time": "2010-01-01/2016-08-01/P2Y2M2D",
                "filter": "highway=residential and type:way"
            }
        },
        {
            "method": "GET",
            "path": "/metadata"
        }
    ]
}
```

Reply:
Hm.. I guess we'd have to add then more info what exactly can/should go into these examples and how one can contribute there (not in the PR template, but in some readme for example). At least for me this is not clear so far.

Reply:
I made a proposal in 79a85fa87be689d034a59b3cb1c73fb59b82494e which might address this issue.

Reply:
Yes looks good to me 👍